scripts/installer.sh

```diff
@@ -7,7 +7,7 @@ if [ $UID -ne 0 ]; then
     exit 13
 fi
 
-ARCHIVE_URL=$(curl -sSL https://api.github.com/repos/thombashi/sqlitebiter/releases/latest | jq -r '.assets[].browser_download_url' | \grep deb)
+ARCHIVE_URL=$(curl -sSL https://api.github.com/repos/thombashi/sqlitebiter/releases/latest | jq -r '.assets[].browser_download_url' | \grep bionic_amd64.deb)
```

Comment:
Ubuntu codename and architecture would be preferable to extract from the execution environment.

Reference: 
https://github.com/thombashi/sqlitebiter/blob/d0b5fa2d8a422b4834a37dd710724f6e4b19aedd/scripts/build_linux_package.sh#L14

https://github.com/thombashi/sqlitebiter/blob/d0b5fa2d8a422b4834a37dd710724f6e4b19aedd/scripts/build_linux_package.sh#L14https://github.com/thombashi/sqlitebiter/blob/d0b5fa2d8a422b4834a37dd710724f6e4b19aedd/scripts/build_linux_package.sh#L51